functions/go/annotate-mutations/README.md

```diff
@@ -0,0 +1,75 @@
+# annotate-mutations
```

Comment:
This name seems generic. Can this be `annotate-apply-time-mutations` ?

Reply:
fine with me, just longer to type.

Reply:
will change